src/compiler/crystal/tools/doc/templates.cr

```diff
@@ -1,6 +1,32 @@
 require "ecr/macros"
 
 module Crystal::Doc
+  SVG_DEFS = <<-SVGS
+<div class="hidden">
```

Comment:
Minor: This `<div>` is unnecessary, `<svg class="hidden">` is better.

Reply:
Reasoning was if we want to add other SVGs, they would all be hidden.